Core/HLE/sceHprm.cpp

```diff
@@ -44,6 +44,16 @@ static u32 sceHprmIsRemoteExist() {
 	return 0;
 }
 
+static u32 sceHprmRegisterCallback() {
+	DEBUG_LOG(HLE, "sceHprmRegisterCallback()");
```

Comment:
These are unlikely to be called repeatedly, I think we should log these two (and maybe other ones here too) as:

ERROR_LOG(HLE, "UNIMPL %s", __FUNCTION__);